Codegen/test/Codegen.Test.csproj

```diff
@@ -86,13 +86,19 @@
     <None Include="resources\input\**">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="resources\input\TypeCobol\Global_Storage\GlobalStorage2.rdz.tcbl">
```

Comment:
Not needed, could you revert changes in this file please ? The new files will be included thanks to the wildcards.

Reply:
yes